src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -592,665 +592,480 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
   };
 
   return (
-    <OakBox
-      $position="relative"
-      data-testid="lot-picker"
-      $zIndex={
-        (isMobileLotPickerModalOpen || showPhases) && isMobile ? 300 : 101
-      }
-      $maxWidth="all-spacing-23"
-      $borderRadius="border-radius-s"
-      $borderColor={showSubjects || showPhases ? "transparent" : "black"}
-      $ba="border-solid-m"
-    >
-      <OakFlex
+    <OakBox aria-describedby="Choose a curriculum" role="group">
```

Comment:
Sorry, I think this should have been `aria-labelledby` - my bad. Also the value of either attribute is an ID pointing at the jaunty angle label field, not a text string.

Reply:
e.g. `aria-labelledby="choose-curriculum-label"`